sklearn/linear_model/_coordinate_descent.py

```diff
@@ -2360,6 +2360,10 @@ class MultiTaskLassoCV(RegressorMixin, LinearModelCV):
     n_iter_ : int
         number of iterations run by the coordinate descent solver to reach
         the specified tolerance for the optimal alpha.
+    
```

Comment:
```suggestion

```
This will fix your first lint error
```
sklearn/linear_model/_coordinate_descent.py:2363:1: W293 blank line contains whitespace
```